content/en/agent/troubleshooting/agent_check_status.md

```diff
@@ -147,4 +147,3 @@ sudo journalctl -u dd-agent.service
 [1]: /integrations/
 [2]: /agent/troubleshooting/send_a_flare/
 [3]: https://github.com/DataDog/datadog-agent/blob/master/docs/agent/changes.md#service-lifecycle-commands
-[4]: /integrations/
```

Comment:
Please revert this change. Removing the link breaks a link above, in the Linux section, on the Agent v5 tab.